openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
Not at all.

That is the main logic of the whole tenant create/update. It reacts to the response codes from POST N and does the things it needs to to PUT or DELETE etc.. 

Without that you're reimplementing the init logic..